drools-wb-webapp/src/main/resources/org/drools/workbench/DroolsWorkbench.gwt.xml

```diff
@@ -3,6 +3,8 @@
     "http://www.gwtproject.org/doctype/2.7.0/gwt-module.dtd">
 <module>
 
+  <inherits name="com.google.gwt.logging.Logging"/>
```

Comment:
@manstis yes slf4j is available client-side (simply inject a logger) and preferred: http://docs.jboss.org/errai/4.0.0.Beta1/errai/reference/html_single/#sid-74908675_Logging-ClientSideSetup /cc @Rikkola @mbarkley